pact/namespaces/ns.repl

```diff
@@ -5,6 +5,7 @@
  , 'ns-operate-keyset: ["operate"]
  , 'ns-genesis-keyset: { "keys": [], "pred": "="} })
 
+(load "v1/ns.pact")
```

Comment:
This is a little confusing, as presumably it's to show that v1 can be upgraded, but it doesn't cover governance/upgrade for v2. Suggest we 
- document the upgrade test (just a comment is fine)
- exercise v2 governance: fail to upgrade without admin sigs, succeed upgrade with. I think just loading v2 over itself should be fine.

Reply:
Sounds good - I'll add.